lib/views/schedule/schedule.dart

```diff
@@ -12,59 +14,48 @@ class Schedule extends StatelessWidget {
     return Scaffold(
       appBar: AppBar(
         title: const Text('Livestream Schedule'),
-        backgroundColor: ThcColors.darkBlue,
         actions: [
-          Visibility(
-            visible: userType.isAdmin,
-            child: IconButton(
-              icon: const Icon(Icons.add), // Choose your desired icon
-              onPressed: () {
-                // Add your onPressed callback function here
-                // This function will be called when the icon button is clicked
-                navigator.push(const ManageSchedule());
-              },
+          if (userType.isAdmin)
+            IconButton(
+              icon: const Icon(Icons.add),
+              onPressed: () => navigator.push(const ManageSchedule()),
```

Comment:
`Visibility` is nice for when you want something to appear/disappear.

We can avoid building an invisible button by using an if-element here instead.

Reply:
Good to know, thanks!